cmd/cp-main.go

```diff
@@ -377,11 +371,30 @@ func doPrepareCopyURLs(ctx context.Context, session *sessionV8, cancelCopy conte
 	return
 }
 
+func printCopyURLsError(cpURLs *URLs) {
+
+	// Print in new line and adjust to top so that we
+	// don't print over the ongoing scan bar
+	if !globalQuiet && !globalJSON {
+		console.Eraseline()
+	}
+
+	if strings.Contains(cpURLs.Error.ToGoError().Error(),
+		" is a folder.") {
+		errorIf(cpURLs.Error.Trace(),
+			"Folder cannot be copied. Please use `...` suffix.")
+	} else {
```

Comment:
Drop `else`. We always print an error and exit above AFAICT.

Reply:
If we drop the else then it will print 2x error outputs. the errorIf does not exit. I think it's the fatalIf that exits